index.html

```diff
@@ -57,5 +59,204 @@ <h1>
         </ul>
       </main>
     </div>
+
+    <!-- Toolbar-Headline Section Ends -->
+
+    <!-- Work Section Starts -->
+
+    <!-- Project 1 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-1.svg" alt="" />
+      </div>
+      <div>
+        <p class="snapshot-title">Tonic</p>
+        <div class="client-info">
+          <p class="client">CANOPY</p>
+          <div class="circle"></div>
+          <p>Back End Dev</p>
+          <div class="circle"></div>
+          <p>2015</p>
+        </div>
+        <p class="project-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="used-language">
+          <div class="language">html</div>
+          <div class="language">css</div>
+          <div class="language">javascript</div>
+        </div>
+        <div href="#" class="see-project"><a>See Project</a></div>
+      </div>
+    </div>
+
+    <!-- Project 2 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-2.svg" alt="" />
```

Comment:
- I believe it will be better to wrap all your project cards inside a section tagas this represents a generic stand alone container/section in an html file. 
- Can we add something like this 👉🏽 `<section class="all-project-container">`
- What do you think about this?

Reply:
I think it'll be good. After wrapping should i add any more style?

Reply:
- No, your styling so far is great. But if the new changes in html file requires you to modify your style, then i think you can go ahead and modify it. It wouldn't take you so much time, i trust you 💪🏽

Reply:
Thanks. done

Reply:
I appreciate you mate and thanks for the support!